node_helper.js

```diff
@@ -345,11 +345,41 @@ module.exports = NodeHelper.create({
         if (bytes.r[0].test(data) && bytes.r[1].test(data)) {
             this.sendSocketNotification('BYTES', bytes.a);
         } else if (/(WAKE)/g.test(data) && /(UP)/g.test(data)) {
-            exec('/opt/vc/bin/tvservice -p && sudo chvt 6 && sudo chvt 7', null);
-            this.hdmi = true;
+						switch(this.config.mode.toLowerCase()){
+							case 'pi':
+								exec('/opt/vc/bin/tvservice -p && sudo chvt 6 && sudo chvt 7', null);
+								this.hdmi = true;
```

Comment:
you can remove the hdmi module variable, it's not used anymore

Reply:
ok, just wrappered the existing code.. no change..

Reply:
i am using the develop branch 

```
    /** @member {boolean} hdmi - Flag to indicate hdmi output state. */
    hdmi: true,
```

Reply:
yes I searched in the project, looks like the variable is only assigned in two spots, but is never used, so it can be removed

Reply:
changed